src/include/distributed/worker_protocol.h

```diff
@@ -45,6 +45,8 @@
 /* Defines used for fetching files and tables */
 /* the tablename in the overloaded COPY statement is the to-be-transferred file */
 #define TRANSMIT_REGULAR_COMMAND "COPY \"%s\" TO STDOUT WITH (format 'transmit')"
```

Comment:
Do we have any callers of `TRANSMIT_REGULAR_COMMAND `? Maybe remove `TRANSMIT_WITH_USER_COMMAND ` and add `user '%s'` to `TRANSMIT_REGULAR_COMMAND` for future callers?

Reply:
Removing the `TRANSMIT_REGULAR_COMMAND`. I kind of like the `WITH_USER` part since it's a bit dangerous if you forget.